pkg/operator/etc/config.go

```diff
@@ -25,6 +25,7 @@ type Config struct {
 	MetricsBindAddress                           string         `env:"OPERATOR_METRICS_BIND_ADDRESS" envDefault:":8080"`
 	MetricsFindingsEnabled                       bool           `env:"OPERATOR_METRICS_FINDINGS_ENABLED" envDefault:"true"`
 	MetricsVulnerabilityId                       bool           `env:"OPERATOR_METRICS_VULN_ID_ENABLED" envDefault:"true"`
+	MetricsExposedSecretInfo                     bool           `env:"OPERATOR_METRICS_EXPOSED_SECRET_INFO_ENABLED" envDefault:"true"`
```

Comment:
`MetricsExposedSecretInfo` should be false (optional) by default right ?

Reply:
ahh, good point 👍🏼 

I copied over `MetricsVulnerabilityId` which looks like is set to default true as well, should I change it to false as well?